docs/index.rst

```diff
@@ -1,11 +1,9 @@
 JAX reference documentation
 ===========================
 
-Composable transformations of Python+NumPy programs: differentiate, vectorize,
-JIT to GPU/TPU, and more.
-
-For an introduction to JAX, start at the
-`JAX GitHub page <https://github.com/google/jax>`_.
+JAX is Autograd_ and XLA_, brought together for high-performance machine learning research.
```

Comment:
Silly question: maybe "numerical computing research" > "machine learning computing research"? I saw the former somewhere maybe in the earlier JAX docs 🤔 It may also be more inclusive, given the use cases popping up potentially even outside of "ML for X".

Reply:
Done!